src/cockpit/packages.py

```diff
@@ -227,22 +259,31 @@ def get_content_security_policy(self, origin):
     def serve_file(self, path, channel):
         filename = self.negotiate_file(path, channel.headers)
 
-        if filename is None:
+        if filename:
```

Comment:
`... is not None` lol

I don't think it matters in this case, because there's no way we found an empty filename as a variant, but seriously :sweat_smile: